magnolia/shared/src/test/scala/zio/config/magnolia/AutomaticConfigDescriptorSpec.scala

```diff
@@ -58,7 +58,8 @@ object AutomaticConfigTestUtils {
 
   final case class Aws(region: String, security: Credentials)
 
-  final case class DbUrl(value: String) extends AnyVal
+  // Scala3 didn't like final case class DbUrl(value: String) extends AnyVal
```

Comment:
What do you mean by "didn't like"?

Reply:
Hey Jules, thanks for coming up here and review.
Yes, scala3 gave out a unreasonable error during implicit inline resolution when giving `extends AnyVal`. End of the day, zio-config never had any special behaviour for `AnyVal`. Users, who need a new type is encouraged to define custom `Derivation` than injecting more magic into the library.

<img width="990" alt="image" src="https://user-images.githubusercontent.com/7448613/124430898-1409c080-ddb3-11eb-8a99-dfc39a795074.png">

Reply:
Il need a macros to summon values explicitly for AnyVal